src/lib/Limitation/Templating/LimitationBlockRenderer.php

```diff
@@ -160,4 +148,15 @@ protected function getValueFallbackBlockParameters(Limitation $limitation, array
 
         return $parameters;
     }
+
+    private function getLimitationValueResources(): array
+    {
+        $resources = $this->configResolver->getParameter('limitation_value_templates');
+
+        usort($resources, function ($a, $b) {
+            return $b['priority'] - $a['priority'];
```

Comment:
AFAICS this is another great possibility to use useless spaceship operator! Don't miss it :exclamation: